server/routes/organizations/gigs/gig-controller.js

```diff
@@ -3,18 +3,32 @@ module.exports = function(app){
   var Gig = models.Gig;
   var UserGigs = models.UserGigs;
   var User = models.User;
+  var Location = models.Location;
 
   var gigController = {
 
     getGigs: function (req, res){
-      var organizationId = req.organization_id;
-      Gig.getGigs(organizationId).then(function (gigs) {
+      var info = {
+        organizationId: req.organization_id,
+        startDate: req.query.startDate,
```

Comment:
I'm not a fan of the API requiring the same startDate and endDate in the url to specify a single day. Could we abstract this logic and just include a date query and then let the server handle the logic that says if the query is for date, set the startDate and endDate  equal to date and includeStaff = true and then carry forward from there.